components/vocabularies/hgnc-remote/api/src/main/java/org/phenotips/vocabulary/internal/RemoteGeneNomenclature.java

```diff
@@ -262,6 +265,12 @@ public VocabularyTerm getTerm(String id)
     }
 
     @Override
+    public List<VocabularyTerm> search(String input, String category, int maxResults, String sort, String customFilter)
```

Comment:
This wouldn't be needed if the default implementation is written in the interface.